pkg/client/clientset/versioned/typed/openebs/v1alpha1/generated_expansion.go

```diff
@@ -15,6 +15,10 @@ limitations under the License.
 */
 package v1alpha1
 
+type CStorPoolExpansion interface{}
```

Comment:
What are these object with suffix Expansion?

Reply:
It is generated by autogen. It reads the type struct name and does some string processing and create new variables.